db/migrate/20180521172343_create_offers.rb

```diff
@@ -0,0 +1,14 @@
+class CreateOffers < ActiveRecord::Migration[5.1]
+  def change
+    create_table :offers do |t|
+      t.references :product
+      t.text :description
+      t.string :title
+      t.integer :suggested_price
+      t.string :status
```

Comment:
You need to add default status (default: proposed)

Reply:
Done